Sources/main.swift

```diff
@@ -0,0 +1,75 @@
+import HTTPServer
+import Router
+import JSON
+
+func getIdParameter(from request: Request) -> Int {
+    guard let idString = request.pathParameters["id"] else { return 0 }
+    return Int(idString) ?? 0
+}
+
+let router = Router { route in
+
+    var todos = [JSON?]()
+    let corsHeader = Headers(["access-control-allow-origin" : Header(stringLiteral: "*")])
```

Comment:
`stringLiteral` means that you can just say `"*"` instead of `Header(stringLiteral: "*")` :smile: